presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/environment/SinglenodeMysql.java

```diff
@@ -70,7 +71,8 @@ private DockerContainer createMySql()
                 .withEnv("MYSQL_DATABASE", "test")
                 .withCommand("mysqld", "--port", Integer.toString(MYSQL_PORT))
                 .withStartupCheckStrategy(new IsRunningStartupCheckStrategy())
-                .waitingFor(forSelectedPorts(MYSQL_PORT));
+                .withHealthCheckCommand(format("mysqladmin ping -h 127.0.0.1 -P %d -u $MYSQL_USER --password=$MYSQL_PASSWORD", MYSQL_PORT))
```

Comment:
Is it better than running a query (can be using `mysql` cli)?

Reply:
```
"mysqladmin ping" attempts to connect to the chosen MySQL server. The actual server can be specified with the -h option (e.g. mysqladmin ping -h db.example.com), defaulting to localhost.

If the server responds (even with an access denied or similar message), the server is assumed to be alive and "mysqladmin ping" exits with code 0.
```